ArchiSteamFarm/Bot.cs

```diff
@@ -534,8 +534,17 @@ public sealed class Bot : IAsyncDisposable {
 					continue;
 				}
 
+				uint maxSetsAllowed = Trading.MaxItemsPerTrade - (uint) result.Count;
```

Comment:
`Trading.MaxItemsPerTrade` is `byte`, change `maxSetsAllowed` to `byte` and correct the cast.

Reply:
Since you never go above the value, it's fine to do all remaining operations on `byte` as well. The final result of `Math.Min()` can be safely casted to `byte`.